reference/conanfile/tools/gnu/pkgconfigdeps.rst

```diff
@@ -94,3 +94,144 @@ flags and require relations.
 Additionally, a *<PKG-NAME>.pc* is generated to maintain compatibility for consumers with recipes that start supporting components. This
 *<PKG-NAME>.pc* file will declare all the components of the package as requires while the rest of the fields will be empty, relying on
 the propagation of flags coming from the components *<[PKG-NAME]-[COMP-NAME]>.pc* files.
+
+
+Names and aliases
+++++++++++++++++++
+
+Aliases are available since: `1.43.0 <https://github.com/conan-io/conan/releases>`_
+
+By default, the ``*.pc`` file names will follow the rules defined below:
+
+* For packages, the own package name, e.g., package ``zlib/1.2.11`` -> ``zlib.pc``.
+* For components, the package name as namespace plus the own component one, package ``openssl/3.0.0`` and component ``openssl::crypto`` -> ``openssl-crypto.pc``.
+
+Given that, it's also possible to change easily that behavior thanks to ``pkg_config_name`` and ``pkg_config_aliases`` properties.
+
+For instance, ``openssl/3.0.0``` recipe has these ``pkg_config_name`` properties already declared:
```

Comment:
```suggestion
```

Reply:
I don't get why you're removing this line

Reply:
I just saw your comment below 😁